azure/functions/decorators/function_app.py

```diff
@@ -837,6 +842,136 @@ def decorator():
 
         return wrap
 
+    def cosmos_db_trigger(self,
+                          arg_name: str,
+                          connection: str,
+                          database_name: str,
+                          container_name: str,
+                          lease_connection: Optional[str] = None,
+                          lease_database_name: Optional[str] = None,
+                          lease_container_name: Optional[str] = None,
+                          create_lease_container_if_not_exists: Optional[
+                              bool] = None,
+                          leases_container_throughput: Optional[int] = None,
+                          lease_container_prefix: Optional[str] = None,
+                          feed_poll_delay: Optional[int] = None,
+                          lease_acquire_interval: Optional[int] = None,
+                          lease_expiration_interval: Optional[int] = None,
+                          lease_renew_interval: Optional[int] = None,
+                          max_items_per_invocation: Optional[int] = None,
+                          start_from_beginning: Optional[time] = None,
+                          start_from_time: Optional[time] = None,
+                          preferred_locations: Optional[str] = None,
+                          data_type: Optional[
+                              Union[DataType, str]] = None,
+                          **kwargs: Any) -> \
+            Callable[..., Any]:
+        """The cosmos_db_trigger decorator adds :class:`CosmosDBTrigger`
+        to the :class:`FunctionBuilder` object
+        for building :class:`Function` object used in worker function
+        indexing model. This decorator will work only with extension bundle 4.x
+        and above. For additional details, please refer
+        https://github.com/Azure/azure-functions-python-worker/issues/1222.
+        This is equivalent to defining CosmosDBTrigger in the function.json
+        which enables function to be triggered when CosmosDB data is changed.
+        All optional fields will be given default value by function host when
+        they are parsed by function host.
+
+        Ref: https://aka.ms/azure-function-binding-cosmosdb-v2
```

Comment:
You might need another aka.ms link which pivots to
<img width="806" alt="image" src="https://user-images.githubusercontent.com/1047040/234770557-201e8af3-1166-4563-8020-f0240f754d66.png">

Reply:
Similar comments to the other decorators as well.